tensorflow_probability/python/layers/distribution_layer.py

```diff
@@ -47,7 +47,7 @@
 from tensorflow_probability.python.distributions import variational_gaussian_process as variational_gaussian_process_lib
 from tensorflow_probability.python.internal import distribution_util as dist_util
 from tensorflow_probability.python.layers.internal import distribution_tensor_coercible as dtc
-from tensorflow_probability.python.layers.internal import tensor_tuple as tensor_tuple
+from tensorflow_probability.python.layers.internal import tensor_tuple as tens_tuple
```

Comment:
please leave this as it was

Reply:
Rolled back this change, although it caused pylint error.